### ISSUE TITLE
Feature request: Custom message in runner

### ISSUE THREAD

**Issue by dbkaynor** [a85/POSTMan-Chrome-Extension#4175]
Is it possible to display a custom message in runner? As an example if a test fails I would like runner (or newman) to display a message telling the operator what needs to be done to correct the problem. I am able to use the following code but I would like it be display in a highlight color such as orange.

pm.test("**** If the following tests fail a credit card must be added to the account ****",true);

Also is it possibe to abort the test on a failure?

## App Details:

Postman for Windows 10 64 bit
Postman v5.5.2

**Comment by sivcan** [comment]
Hi @dbkaynor, for your first suggestion, I'll put it up as a feature request.

For the second one I needed some clarification 
Are you saying that when a test fails, you want to stop the runner from running more requests ?

**Comment by dbkaynor** [comment]
Yes, when a test fails, I want to abort any following tests. Also, I would like to be able to be able to be able to conditionally run some tests and skip others. For example, I have some tests that need to be run daily and other only once a week.
